### DISCUSSION TITLE
On build show html and js gzipped size sorted largest to smallest

### DISCUSSION THREAD

**Discussion by KyleAMathews** [gatsbyjs/gatsby#430] [Ideas / Feature Requests]
Perhaps store these by git hash so can compare sizes overtime. Also warn when a particular page grows say more than 10% over a previous commit.

**Comment by Chuloo** [comment]
There hasn't been any activity on this issue recently. Due to the high number of incoming GitHub notifications, we have to clean some of the old issues as many of them have already been resolved with the latest updates.

Please make sure to update to the latest Gatsby version and check if that solves the issue. Let us know if that works for you by adding a comment 👍

**Comment by KyleAMathews** [comment]
Let's leave this one open — it's still a feature that we're probably going to add.

**Comment by alex-saunders** [comment]
Is this still a requested feature and is it a beginner-friendly issue? Looks like something I'd like to work on but don't want to be too in-over-my-head!

**Comment by jlengstorf** [comment]
@alex-saunders My gut says this is going to start simple and then take a turn for the complex pretty fast. There are a lot of internals involved (e.g. how bundles are named) that will probably require extra context. If you're interested in a Gatsby deep dive, the Inkteam can help you find the relevant areas, but I wouldn't call this one beginner-friendly.

**Comment by alex-saunders** [comment]
@jlengstorf No problem, I'm looking for more beginner-friendly issues so think I'll avoid this one! Are you able to point me to any issues that might be more appropriate to work on? (no problem if there's none available atm!)

**Comment by jlengstorf** [comment]
@alex-saunders We've got a good chunk of issues [tagged with good first issue](https://github.com/gatsbyjs/gatsby/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) that are a good place to start. There are some [front-end focused issues on the swag store](https://github.com/gatsbyjs/store.gatsbyjs.org/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) that would be beginner friendly as well.

Feel free to claim one, and tag me if you have questions!

**Comment by mathyouf** [comment]
@jlengstorf 

> If you're interested in a Gatsby deep dive, the Inkteam can help you find the relevant areas, but I wouldn't call this one beginner-friendly.

I'm interested in a Gatsby deep-dive, where can I find info about this?

**Comment by jlengstorf** [comment]
@11fishem we have a "behind the scenes" section in our docs that goes deep. This is probably the best starting point: https://www.gatsbyjs.org/docs/behind-the-scenes/

Happy spelunking!
